addons/mail/models/mail_message.py

```diff
@@ -85,6 +85,7 @@ def default_get(self, fields):
     # content
     subject = fields.Char('Subject')
     date = fields.Datetime('Date', default=fields.Datetime.now)
+    edit_date = fields.Datetime('last edited DateTime', readonly=1)
```

Comment:
To keep model as simple as possible, you may use write_date instead. As messages are never edited this should be sufficient in most use cases and an unnecessary column on an heavily used model.

Reply:
As discussed earlier, In the case of starred message write_date is also updated. So we won't get the proper write_date in the case to put the `edited` at the end of the message.

Reply:
We have several millions of mail.message in DB, adding a column when 95% of messages are not edited is globally wasting space. Editing message is something you globally don't do except the discuss use case. I still think write_date is sufficient.

The other way round would be to store extra information on another table (edit_date, star status, ...). That way we lessen noise on a big table.